packages/docs/src/locales/en/en.json

```diff
@@ -3038,18 +3038,15 @@
       "title": "Tree shaking",
       "description": "If you plan to use only several components in your application and thus reduce the size of your bundle, you can use the Vuestic ESM build. Webpack (vue-cli) and Vite do that automatically.",
       "example": {
-        "title": "First, you don't need to use `VuesticPlugin` since it registers all the `vuestic` components globally. Instead, you can import specific components from the `vuestic-ui` package.",
-        "footer": "Alternatively, you can import them directly in your `.vue` files."
+        "title": "First, you don't need to use `createVuestic` since it registers all the `vuestic` components globally. We have `createVuesticEssential` instead, which register only essential plugins. You can specify components to declare globally. Or you can import them later.",
```

Comment:
core vs essentials vs full we discussed before.
We can leave essentials and full in docs, but all 3 should be depicted in notion so that our strategy is clear.